test/integration/02-user-wallet/02-send-onchain.spec.ts

```diff
@@ -46,30 +58,41 @@ const date = Date.now() + 1000 * 60 * 60 * 24 * 8
 jest.spyOn(global.Date, "now").mockImplementation(() => new Date(date).valueOf())
 
 let initialBalanceUser0
-let userWallet0, userWallet3, userWallet11, userWallet12 // using userWallet11 and userWallet12 to sendAll
+let userWallet0
 
-let wallet0: WalletId
-let user0: UserId
+let walletId0: WalletId
+let walletId1: WalletId
+let walletId3: WalletId
+// using walletId11 and walletId12 to sendAll
+let walletId11: WalletId
+let walletId12: WalletId
+let userId0: UserId
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const { sendNotification } = require("@services/notifications/notification")
 
 beforeAll(async () => {
   userWallet0 = await getAndCreateUserWallet(0)
```

Comment:
I would try to delete `userWallet0`. this would allow the deletion of `getAndCreateUserWallet` method.

Reply:
if you need the user object associated with user0, there is a new method for it `getUserTypeByTestUserIndex`. it'll give you the `userType` object, equivalent to `userWallet0.user`.

Reply:
Updated and `getAndCreateUserWallet` was removed https://github.com/GaloyMoney/galoy/pull/911/commits/7a9d01b554c4b98e50409140ed6c3014f859763d